cloudfront/wellcomelibrary.org/edge-lambda/apiRedirects.csv

```diff
@@ -0,0 +1,14 @@
+sourceUrl,targetUrl
+wellcomelibrary.org/iiif/b28047345/manifest,https://iiif.wellcomecollection.org/presentation/v2/b28047345
+wellcomelibrary.org/iiif/collection/b18031511,https://iiif.wellcomecollection.org/presentation/v2/b18031511
+wellcomelibrary.org/iiif/b18031511-2/manifest,https://iiif.wellcomecollection.org/presentation/v2/b18031511_0003
+wellcomelibrary.org/item/b18031511,https://wellcomecollection.org/works/ysmqsfhg
```

Comment:
Without having looked further – I assume that `ysmqsfhg` is the work corresponding to b18031511.

What if I look up an ID that can't be found in the catalogue API? e.g. an old link to a now-deleted item.

Ditto other URLs, I guess.

Reply:
Try it: http://stage.wellcomelibrary.org/item/blarg (should 404 you deliberately on a works page).

It might be a bit unclear but these are ~smoke tests to ensure the deployed code is working as expected, so they won't have deliberately failing test cases.